streams/stable.json

```diff
@@ -185,92 +185,16 @@
             "images": {
                 "aws": {
                     "regions": {
```

Comment:
looks like we lost aws replication somehow

Reply:
good catch Dusty!

Reply:
should be fixed now.